VS/VehicleSignals.rdf

```diff
@@ -1023,6 +1025,7 @@
 	
 	<owl:Class rdf:about="&auto-vs;CoolantTemperatureSensor">
 		<rdfs:subClassOf rdf:resource="&auto-vs;VehicleSensor"/>
+		<rdfs:label xml:lang="en">Coolant Temperature Sensor</rdfs:label>
```

Comment:
Note that https://github.com/edmcouncil/fibo/blob/master/ONTOLOGY_GUIDE.md#naming-and-labeling-conventions requires all labels be lower case.